src/assets/toolkit/styles/components/sky.css

```diff
@@ -238,15 +268,27 @@
  * 1. At large enough screens, make the menu _always_ visible.
  * 2. Remove the bump (since it won't be above anything at this point).
  * 3. Reset the background.
- * 4. Reset the order.
+ * 4. To align right, this now needs to be last.
+ * 5. Constraining the width keeps the last three links on a second line to avoid orphans.
  */
 
 @media (--Sky-nav-expanded-viewport) {
   .Sky-nav-menu {
     display: block; /* 1 */
     padding-bottom: 0; /* 2 */
     background: transparent; /* 3 */
-    order: 0; /* 4 */
+    order: 2; /* 4 */
```

Comment:
In the markup, the source order is:

1. Controls
2. Logo
3. Nav

So why do we change their `order` at all? Because at small screens, the `nav` must be _above_ the other two elements, and the "Menu" toggle needs to be to the _right_ of the logo.

But at large screens, neither is the case. The "menu" toggle isn't even visible, and the "nav" is fine exactly where it would be naturally.

With this in mind, you _should_ be able to set the nav's `order` to `1` (same as the controls), or you could even reset the `order` of controls _and_ nav (both `0`) because their natural order is just fine after that breakpoint.